docs/versioned_docs/version-2.11.0/introduction/benefits.mdx

```diff
@@ -0,0 +1,124 @@
+    For example, developers and DevOps architects can use standard Kubernetes
+    skills and utilities to configure, use, and manage the persistent storage
+    needs.
+  </p>
+
+![Cloud Native Storage Icon](../assets/b-cn.svg)
+
+</TwoColumn>
+
+Some key aspects that make OpenEBS different compared to other traditional storage solutions:
+
+- Built using the _micro-services architecture_ like the applications it serves. OpenEBS is itself deployed as a set of containers on Kubernetes worker nodes. Uses Kubernetes itself to orchestrate and manage OpenEBS components.
+- Built completely in userspace making it highly portable to run across _any OS/platform_.
+- Completely intent-driven, inheriting the same principles that drive the _ease of use_ with Kubernetes.
+- OpenEBS supports a range of storage engines so that developers can deploy the storage technology appropriate to their application design objectives. Distributed applications like Cassandra can use the _LocalPV_ engine for lowest latency writes. Monolithic applications like MySQL and PostgreSQL can use _Mayastor built using NVMe and SPDK_ or _cStor based on ZFS_ for resilience. Streaming applications like Kafka can use the NVMe engine [Mayastor](https://github.com/openebs/Mayastor) for best performance in edge environments.
+
+### Avoid Cloud Lock-in
+
+<TwoColumn>
+  <p>
+    Even though Kubernetes provides an increasingly ubiquitous control plane,
+    concerns about data gravity resulting in lock-in and otherwise inhibiting
+    the benefits of Kubernetes remain. With OpenEBS, the data can be written to
+    the OpenEBS layer - if cStor, Jiva or Mayastor are used - and if so OpenEBS
+    acts as a data abstraction layer. Using this data abstraction layer, data
+    can be much more easily moved amongst Kubernetes environments, whether they
+    are on premise and attached to traditional storage systems or in the cloud
+    and attached to local storage or managed storage services.
+  </p>
+
+![Avoid Cloud Lock-in Icon](../assets/b-no-lockin.svg)
+
+</TwoColumn>
+
+### Granular Policies Per Stateful Workload
+
+<TwoColumn>
+    <p>
+        One reason for the rise of cloud native, loosely coupled architectures is that they enable loosely coupled teams. These small teams are enabled by cloud native architectures to move faster, free of most cross functional dependencies thereby unlocking innovation and customer responsiveness. OpenEBS also unlocks small teams by enabling them to retain their autonomy by virtue of deploying their own storage system. Practically, this means storage parameters are monitored on a workload and per volume basis and storage policies and settings are declared to achieve the desired result for a given workload. The policies are tested and tuned, keeping only the particular workload in mind, while other workloads are unaffected. Workloads - and teams - remain loosely coupled.
+    </p>
+
+![Workload Granular Policies Icon](../assets/b-granular.svg)
```

Comment:
Is there any pattern you're following with the image names?

Reply:
I believe no